app/src/main/res/drawable/activity_icon.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
For now, it's okay because you are implementing basic things but try to create smaller commits. In this case, you may have created a merge request with resources (icon, image, strings, ...) and another one with code. This way there is less noise and merge request is easier to review.